assets/js/editor.js

```diff
@@ -129,6 +129,28 @@ $gp.editor = (
 					} else {
 						$gp.editor.save( target.parents( 'tr.editor' ).find( 'button.ok' ) );
 					}
+				} else if ( 13 === e.keyCode && e.ctrlKey ) { // Ctrl-Enter = Copy original.
+					target = $( e.target );
+
+					original = $( '.editor:visible' ).find( '.original' );
+
+					target.val( original.text() );
+				} else if ( 107 === e.keyCode && e.ctrlKey ) { // Ctrl-+ = Approve.
```

Comment:
`Ctrl-+` zooms in the webpage in most browsers in Windows, see [Windows > Webpage shortcuts > Make everything on the page bigger](https://support.google.com/chrome/answer/157179)